modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -1429,6 +1454,7 @@ public void close() {
     }
 
     // TODO: IGNITE-17963 Use smarter logic for recipient node evaluation.
+
```

Comment:
What's the meaning of this empty line and why the next javadoc is re-formatted? Was that done by IDEA?

Reply:
yes, I've applied auto-formatting